.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
```suggestion
```

Sorry, but everyone can burn my wallet by simply submitting PRs repeatedly. :(

Reply:
GitHub Actions is free for public repositories, so no worries for this!

You can also enable "Require approval for first-time contributors" in this repository's Actions settings, so actions will not be ran for people without any existing commits in the repo. This can help avoiding useless runs by bots or other weird users.

Reply:
Thanks for the explanation. I didn't know about that.